src/app/ngrx/reducers/episode.reducer.ts

```diff
@@ -1,78 +1,91 @@
-import { ActionTypes, CmsPodcastEpisodePageSuccessAction, CmsPodcastEpisodePageFailureAction, AllActions  } from '../actions';
-import { HalDoc } from 'ngx-prx-styleguide';
+import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
+import { Episode } from './models/episode.model';
+import { ActionTypes, AllActions } from '../actions';
 
-export const EPISODE_PAGE_SIZE = 10;
-
-export interface EpisodeModel {
-  doc?: HalDoc;
-  id: number;
-  seriesId: number;
-  title: string;
-  publishedAt: Date;
-  page?: number;
-  color?: string;
-  feederUrl?: string;
-  guid?: string;
+export interface State extends EntityState<Episode> {
+  // additional entities state properties
+  pagesLoaded: number[];
```

Comment:
we're just keeping track of the current podcasts episode loading/loaded/total state here. This doesn't quite fit with the shape of the state, and depending on how I decide to proceed with getting all the episode data for that nav menu is definitely subject to change. But +1 for composability making these it easier to shift around where we keep track of things.